content/docs/04.5-devx/03-cluster-groups.md

```diff
@@ -101,6 +101,35 @@ To delete a cluster group, select `Delete Cluster form the **Settings** and ente
 * Azure
 * AKS
 * VMWare
+* OpenShift
+
+<Tabs>
+<Tabs.TabPane tab="Configure OpenShift" key="Configure OpenShift">
+
+#### Configure OpenShift 
+<br />
+To deploy a new virtual cluster on OpenShift:
+
+
+1. Select the new Cluster Group and click **Settings**.
+2. Select **Settings** in **Cluster Group Settings**.
+3. In the **Advanced Config** file, locate the section ``Configure SecurityContext of the containers in the VCluster pod``.
+
+4. Comment out these lines: 
+
+    * ``fsgroup``
+    * ``runAsGroup``
+    * ``runAsUser``
+4. Set ``openshift: enable: true``.
+5. Ensure the following parameters are set as follows: 
```

Comment:
This step is a non-step (no action required) so maybe we can leave it out. Or if we include it, we should also mention
```
capabilities:
  drop:
  - all
```